[XAM/XMP] - XMP Improvements

The-Little-Wolf opened the pull request:
- Proper returns for XMPContinue, XMPStop, XMPPause, XMPNext, and XMPPrevious
- left notes
- all playback changes have been put into another commit

has207 commented:
I tried patching this commit and noticed it broke menu bgm in Afro Samurai (it's xam not xmp), jfyi

Sowa95 commented:
Fixes missing audio in [58411446 - Guacamelee!](https://github.com/xenia-canary/game-compatibility/issues/866)

The-Little-Wolf committed:
[XAM/XMP] - XMP Improvements

- Proper returns for XMPContinue, XMPStop, XMPPause, XMPNext, and XMPPrevious
- left notes

The-Little-Wolf committed:
[XAM/XMP] - XMP Improvements

- Proper returns for XMPContinue, XMPStop, XMPPause, XMPNext, and XMPPrevious
- left notes

Sowa95 commented:
No noticeable difference in [454107E9 - The Godfather](https://github.com/xenia-canary/game-compatibility/issues/256)

larryeedwards commented:
I just tried the PR myself and I'm noticing an issue with [4B4E0806 - Tornado Outbreak. ](https://github.com/xenia-canary/game-compatibility/issues/56). By default now, the game's music is entirely muted. Everything else still works fine though.
